modules/reactor/test/tests/meshgenerators/hexagon_mesh_trimmer/assembly_trim.i

```diff
@@ -0,0 +1,55 @@
+[Mesh]
+  final_generator = trim
```

Comment:
it d be better to set the active/inactive.

rn you run a bunch of extra generators and discard the results